### ISSUE TITLE
Is it really necessary to "prepare likelihoods for annotation"?

### ISSUE THREAD

**Issue by droazen** [broadinstitute/gatk#3022]
@davidbenjamin commented on [Sat May 27 2017](https://github.com/broadinstitute/gatk-protected/issues/1112)

In `SomaticGenotypingEngine::callMutations` and `HaplotypeCallerGenotypingEngine::assignGenotypeLikelihoods` there is a line of code after the call is made but before the variant is annotated:
```java
ReadLikelihoods annotationLikelihoods = prepareReadAlleleLikelihoodsForAnnotation(likelihoods...)
```
Is this really necessary?  It seems quite defensible to annotate using the same likelihoods from which the variant call is derived.  As far as Mutect is concerned, the standard will be whether our validations are better or at least no worse without it.

**Comment by davidbenjamin** [comment]
Answer: no.  PR #3243 is under review.

**Comment by davidbenjamin** [comment]
Closed by #3243.
